addons/mail/static/src/core/common/thread.xml

```diff
@@ -102,14 +98,23 @@
 </t>
 
 <t t-name="mail.Thread.jumpPresent">
-    <span t-if="props.showJumpPresent and state.showJumpPresent" class="o-mail-Thread-jumpPresent position-sticky btn btn-link alert alert-info d-flex cursor-pointer align-items-center py-2 m-0" t-att-class="{ 'px-4': !env.inChatWindow, 'px-2': env.inChatWindow, 'top-0': props.order !== 'asc', 'bottom-0': props.order === 'asc' }" role="button" t-on-click="() => this.jumpToPresent()">
-        <span class="small">You're viewing older messages</span>
-        <span class="flex-grow-1"/>
-        <span class="fw-bolder small pe-2">Jump to Present</span>
-        <i class="fa" t-att-class="{ 'fa-caret-up': props.order !== 'asc', 'fa-caret-down': props.order === 'asc' }"/>
+    <span  t-if="props.showJumpPresent and state.showJumpPresent" t-att-class="{
+        'mx-5 rounded-top': !env.inChatWindow and !env.inChatter,
+        'm-0 px-4 position-sticky top-0': env.inChatter
+    }" class="o-mail-Thread-banner d-flex justify-content-between alert alert-primary border-0 rounded-0 mb-0 py-1 cursor-pointer shadow-sm small fw-bold" t-on-click="() => this.jumpToPresent()">
+        <span>You're viewing older messages</span>
+        <span>Jump to Present<i class="ms-2 fa" t-att-class="{ 'fa-caret-up': props.order !== 'asc', 'fa-caret-down': props.order === 'asc' }"/></span>
     </span>
 </t>
 
+<t t-name="mail.Thread.jumpToUnread">
+    <span t-if="props.thread.selfMember?.localMessageUnreadCounter >= 10" class="d-flex cursor-pointer shadow-sm small fw-bold" t-att-class="{'mx-5': !env.inChatWindow}">
```

Comment:
10 looks quite an arbitrary threshold. I think this can be confusing for end-users to guess the trigger of this banner. 

With quick-search, users add channels and chats until they reach 20, and it's quite stable its presence because you usually add more rather than less, and it's very likely to reach this amount frequently in practice.

With this condition on unread, however, this looks quite rare, and when you see it you are likely to mark thread as read soon, and 10 is a lot of messages to have it unread, so it's relatively unlikely to have it again very soon.
In the code it looks easy, but in practice I expect some users to find out this threshold after some days of use and confusion